### PR TITLE
Significant performance improvements when dealing with larger data

### PR DESCRIPTION
**Pull request by magnificus**
…ts called in interface, sped it up

there are more things I found I will fix

### CODE REVIEW

File: src/nodes/datatypes/jsonType.tsx [hunk 1]
```diff
@@ -23,9 +22,6 @@ export class JSONType extends AbstractType {
 
   getInputWidget = (props: JSONTypeProps): any => {
     props.dataType = this;
-    if (typeof props.property.data !== 'string') {
```
Comment (magnificus) [inline src/nodes/datatypes/jsonType.tsx:26]: this was setting the data on the socket itself to a stringified version on widget open, was costing a lot of performance and seemed very strange to me

**Review by fakob — CHANGES_REQUESTED** [review]
The point with the "Info" tab for both graph and node is that you can
* see its current config (serialized view)
* edit and replace it

I understand that it is not practical to serialize the graph or node every time you open the inspector, but then I suggest that we introduce a "load config" button in the "Info" tab, which would serialize the graph or node. Currently the config part of the info tab has lost its meaning as it does not show the current state.

Screenshot 2023-12-11 at 21 34 30

**Comment by magnificus** [comment]
OK, load config button sounds good to me, would you be able to help me with that? will then remove the parameter to serialize "includeSockets"

**Comment by fakob** [comment]
I have now pushed a change where the sourceContent is broken out into its own component and in there I have added a load data button. Please have a look.

Screenshot 2023-12-13 at 23 33 42

**Pull request merged**